src/main/java/mekanism/generators/common/MekanismGenerators.java

```diff
@@ -101,13 +101,13 @@ public void postInit(FMLPostInitializationEvent event)
 		{
 			for(IFuel s : BuildcraftFuelRegistry.fuel.getFuels())
 			{
-				if(!(s.getFluid() == null || GasRegistry.containsGas(s.getFluid().getName())))
+				if(!(s.getFluid() == null || GasRegistry.containsGas(s.getFluid().getFluid().getName())))
 				{
-					GasRegistry.register(new Gas(s.getFluid()));
+					GasRegistry.register(new Gas(s.getFluid().getFluid()));
 				}
 			}
 
-			BuildcraftFuelRegistry.fuel.addFuel(MekanismFluids.Ethene.getFluid(), (int)(240 * general.TO_RF), 40 * Fluid.BUCKET_VOLUME);
+			BuildcraftFuelRegistry.fuel.addFuel(MekanismFluids.Ethene.getFluid(), 240, 40 * Fluid.BUCKET_VOLUME);
```

Comment:
I don't know the intentions with the 240 (600 RF). If I'm reading this right, it's about the generated power per tick. In that case I may think 600RF/t is the right value.

But only @aidancbrady could confirm/deny this value.